### ISSUE TITLE
please fix npm start crash on node 7.7.2

### ISSUE THREAD

**Issue by Evrins** [wmonk/create-react-app-typescript#26]
This issue has been fixed on react-scripts 0.9.5

**Comment by FredrikNoren** [comment]
I'm not sure what problem @Evrins is having but I'm also having trouble with node 7.7.2, getting the following from `yarn start`:

```
yarn start v0.21.3
$ react-scripts-ts start
(node:91149) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): TypeError: "listener" argument must be a function
(node:91149) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
✨  Done in 0.79s.
```

After which it just shuts down the process. I'm on the latest react-script-ts (1.1.6)

**Comment by Evrins** [comment]
@FredrikNoren This is the same error message I got when using current version of  react-script-ts (1.1.6)

but on this [release](https://github.com/facebookincubator/create-react-app/releases/tag/v0.9.5)  create-react-app fixed this problem

**Comment by Evrins** [comment]
Magically fixed on node 7.7.3 maybe a bug on node 7.7.2
